src/lucky/cookies/cookie_jar.cr

```diff
@@ -56,15 +56,15 @@ class Lucky::CookieJar
   end
 
   def get_raw(key : Key) : HTTP::Cookie
-    get_raw?(key) || raise "No cookie with the key: #{key}"
+    get_raw?(key) || raise CookieNotFoundError.new("No cookie with the key: #{key}")
   end
 
   def get_raw?(key : Key) : HTTP::Cookie?
     cookies[key.to_s]?
   end
 
   def get(key : Key) : String
-    get?(key) || raise "No cookie for '#{key}'"
+    get?(key) || raise CookieNotFoundError.new("No cookie for '#{key}'")
```

Comment:
To make the error the same no matter where you call it you can add an initializer that accepts the key and then raise the message in the error class. 

I think we’ve done that in a couple places and it’s worked well